FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,50 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .success((let data, let response)):
+				completion(FeedItemsMapper.map(data, from: response))
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+internal struct FeedItemsMapper {
+	private struct ImageFeedResponse: Decodable {
+		let items: [RemoteFeedImage]
+
+		var feedImages: [FeedImage] {
+			return items.map(\.feedImage)
+		}
+	}
+
+	private struct RemoteFeedImage: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
+
+		var feedImage: FeedImage {
+			FeedImage(id: id, description: description, location: location, url: url)
+		}
+
+		private enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+	}
+
+	internal static func map(_ data: Data, from response: HTTPURLResponse) -> RemoteFeedLoader.Result {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it.
```suggestion
	static func map(_ data: Data, from response: HTTPURLResponse) -> RemoteFeedLoader.Result {
```